examples/automm/tabular_dl/README.md

```diff
@@ -1,28 +1,29 @@
-# Advanced Tabular DL models in AutoMM
+# Advanced models in autogluon.multimodal for tabular data
 
-## 1. Tabular Data
 
-
-### 1.1 Example
+### 1. Run Example
 [`example_tabular.py`](./example_tabular.py) : This example provides a use case for the pure *tabular* data, including pure numerical features and numerical + categorical feartures, with FT_Transformer [1].
 
 To run the example: 
 
 ```python example_tabular.py --dataset_name ad --dataset_dir ./dataset --exp_dir ./result```
-   - `dataset_name` determines which dataset to run the experinments, refers to [Dataset Section](###1.2-Datasets).
+   - `dataset_name` determines which dataset to run the experinments, refers to [Dataset Section](###2.-Datasets).
    - `dataset_dir` is the path to the dataset(s). If the datasets do not present in this path, it will be automatically downloaded.
    - `exp_dir` is the output path to store the weights and loggings. 
-   - `seed` determines the random seed. Default is 0.
+   - `--gpu_id` specifies the GPU to use (optional).
```

Comment:
Let’s be consistent here. The other flags do not have “--“ so we should change it to ‘gpu_id’.